app/services/publish_time_parser.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-class DatetimeParser
+class PublishTimeParser
```

Comment:
Is this renamed due to coupling to schedule with the issues?

I wonder if we could make the issue prefix a property of initialize (and default to schedule since this is the only one) so that we can have DateTimeParser and DateParser as generic classes.

Reply:
@benthorner and I had a fairly lengthy discussion about this earlier and tried exploring a few different options.  Might be worth us having a chat face-to-face tomorrow.